frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -867,10 +867,22 @@ function ReaderRolling:onUpdatePos()
         -- we have set above) to avoid multiple refreshes.
         return true
     end
+
+    if Device:isAndroid() then
+        UIManager:discardEvents(60)
+        Device:setIgnoreInput(true)
+    end
```

Comment:
I would say this discardEvents() is not needed, as UIManager won't get any event until setIgnoreInput(false). Is that right ?

Reply:
Yes, can be deleted.

Reply:
You can also remove the `if Device:isAndroid() then` (as those will anyway make a function call I'd have wanted to avoid :)
Please add a comment on the line with the first one: `-- prevent ANR on Android as the following may block for some time`
(And I don't mind having these really around the getCurrentPos() to make it clear it's just this call that blocks, instead of spanned around with blank lines.)